internal/config/namespace.go

```diff
@@ -125,3 +142,19 @@ func IsManagedAnnotation(k string) bool {
 	}
 	return false
 }
+
+// IsHNCServiceAccount is inspired by isGKServiceAccount from open-policy-agent/gatekeeper.
```

Comment:
Hmm, sorry to ask for this, but it's not looking great to have this function in `config` now that I see it here. Maybe just have a `config.GetHNCNamespace()` function to retrieve the value of the the config. Then `isHNCServiceAccount` can move back to the webhooks package, and it can depend on `config.GetHNCNamespace()`.

This would also let you revert a bunch of changes in other files, which is nice.